Info/function.json

```
@@ -5,7 +5,7 @@
       "type": "httpTrigger",
       "direction": "in",
       "name": "req",
-      "route": "template/info",
+      "route": "/api/v1/info",
```

Be careful that the initial `/` doesn't work on production.
```suggestion
      "route": "api/v1/info",
```